### ISSUE TITLE
Follow up of #15967 - https://p2a.brave.com/ endpoints are not listed in any network traffic tools

### ISSUE THREAD

**Issue by GeetaSarvadnya** [brave/brave-browser#20982]
## Description 

Found while testing https://github.com/brave/brave-browser/issues/15967
`Confirm packets to the https://p3a.brave.com/ and https://p2a.brave.com/ endpoints are similar to what what sent before the change, encoded as a binary protocol buffer, instead of in a human-readable strings like the json enpoint receives.`

Follow the test plan from Follow the test plan from https://github.com/brave/brave-core/pull/11600

## Steps to Reproduce

   1. Open Charles or any other n/w traffic tools
   2. Clean profile 1.36.x
   3. Enable rewards 
   4. View an untargeted ad
   5. P2A endpoint https://p2a.brave.com host is not listed in Charles

## Actual result:

https://p2a.brave.com/ endpoints are not listed in any network traffic tools
## Expected result:
https://p2a.brave.com/ endpoints should be listed in network traffic tools

## Reproduces how often: 

Easy
## Brave version (brave://version info)

<html>
<body>

Brave | 1.36.87 Chromium: 98.0.4758.87 (Official Build) beta (64-bit)
-- | --
Revision | e4cd00f135fb4d8edc64c8aa6ecbe7cc79ebb3b2-refs/branch-heads/4758@{#1002}
OS | Windows 10 Version 21H2 (Build 19044.1503)

</body>
</html>

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards? NA
- Is the issue reproducible on the latest version of Chrome? NA

## Miscellaneous Information:

cc: @rillian @moritzhaller @brave/legacy_qa

**Comment by stephendonner** [comment]
In case it's a helpful datapoint; while verifying on macOS in https://github.com/brave/brave-browser/issues/15967, I retested my steps several times, and in doing so, noticed we send data to `p2.brave.com` after ~8-9 minutes from profile start.

I've had up to 6 of these pings appear in a single run, before (not captured here).

My steps:
1. run Charles Proxy
2. new profile
3. enable Rewards
4. open 5 or more new tabs
5. wait for your 1st ad
6. do nothing and let the P3A/P2A fill in!


example | example
---------|---------
<img width="1787" alt="Screen Shot 2022-02-08 at 9 46 26 PM" src="https://user-images.githubusercontent.com/387249/153129608-f2134367-6f1c-4338-a2c8-63695163fd6c.png"> | <img width="1787" alt="Screen Shot 2022-02-08 at 9 49 46 PM" src="https://user-images.githubusercontent.com/387249/153129921-f6cbdd09-adc2-4e30-969a-b00ca5b587c8.png">

I've found -- depending on user actions I've taken in a given session -- I've had to wait up to ~45 minutes for 36 metrics to roll in.  How long has/have your session(s) been, @GeetaSarvadnya ?

**Comment by GeetaSarvadnya** [comment]
@stephendonner I have waited for ~15 mins and I have got 28 metrics Yesterday and I wasn't getting P2A pings. But today when I tried again I can see 6 P2A pings, Charles. Looks like P2A pings are working as expected. Closing the issue as invalid. @stephendonner Thanks for checking!

P2A pings:

p2a pings | local-state p2a metrics
----|-----
![image](https://user-images.githubusercontent.com/38657976/153229249-790b3543-99e9-4837-9ee2-c8c33d75ae7e.png) | ![image](https://user-images.githubusercontent.com/38657976/153229372-f09dc176-8c40-4476-adf2-dbe8567de8f2.png)
